deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./node_modules/.bin/grunt publish:chrome
```

Comment:
@alex-volga can you please add the commands we do for a release. e.g. bump/dump etc